ccd-adapter/src/test/java/uk/gov/hmcts/cmc/ccd/assertion/CCDInterestAssert.java

```diff
@@ -0,0 +1,36 @@
+package uk.gov.hmcts.cmc.ccd.assertion;
+
+import org.assertj.core.api.AbstractAssert;
+import uk.gov.hmcts.cmc.ccd.domain.CCDInterest;
+import uk.gov.hmcts.cmc.domain.models.Interest;
+
+import java.util.Objects;
+
+public class CCDInterestAssert extends AbstractAssert<CCDInterestAssert, CCDInterest> {
+
+    public CCDInterestAssert(CCDInterest actual) {
+        super(actual, CCDInterestAssert.class);
+    }
+
+    public CCDInterestAssert isEqualTo(Interest interest) {
+        isNotNull();
+
+        if (!Objects.equals(actual.getRate(), interest.getRate())) {
+            failWithMessage("Expected CCDInterest.line1 to be <%s> but was <%s>",
```

Comment:
`CCDInterest.rate` instead of `CCDInterest.line1`

Reply:
done